index.d.ts

```diff
@@ -0,0 +1,34 @@
+type Mode = "binary" | "metric" | "decimal" | "compatibility";
```

Comment:
I've actually added `jedec` as well just now during a push to master.

Reply:
Can do.